MVP Module/Source/Feed Presentation/FeedErrorViewModel.swift

```
@@ -0,0 +1,7 @@
+//
+//  Copyright © 2019 Essential Developer. All rights reserved.
+//
+
+struct FeedErrorViewModel {
+	let errorMessage: String
```

The `errorMessage` could be an optional `String?`, so the Presenter can decide when to show/hide the message.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `feedErrorView.display(.none)` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's often easier to manage state when all changes are centralized.